submissions/mariamPopoola/database/init.sql

```diff
@@ -0,0 +1,61 @@
+-- Database: food_bag
+
+
+-- Table structure for food
+
+CREATE TABLE food (
+    id INT NOT NULL AUTO_INCREMENT,
+    type ENUM('food_category_one','food_category_two', 'food_category_three') INT NOT NULL,
+    PRIMARY KEY (id)
+);
+
+
+
+-- Table structure for vendor_food
+
+CREATE TABLE vendor_food (
+    id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(250) NOT NULL,
+    amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (admin_id ) REFERENCES food (id)
```

Comment:
admin_id should reference table admin (column id)
while food_id should reference table food (column id)